Gemfile

```diff
@@ -15,3 +15,7 @@ if version == "master"
 else
   gem "rails", "~> #{version}.0"
 end
+
+group :test do
+  gem "timecop"
+end
```

Comment:
IMHO, any gem required for test to run should be in the gemspec as a development_dependency. your thoughts?